mockwebserver/src/test/java/okhttp3/mockwebserver/MockWebServerTest.java

```diff
@@ -456,4 +457,22 @@
     RecordedRequest request = server.takeRequest();
     assertEquals("request", request.getBody().readUtf8());
   }
+
+  @Test(expected = IllegalArgumentException.class)
```

Comment:
Same as previous comment, OkHttp tests favour explicit catch and check exception message over annotation expected tests.